### PR TITLE
modify code lines for as_json_dict method in Base class

### PR DESCRIPTION
**Pull request by HwangWonYo**
Improved in a more intuitive and easier to understand direction

### CODE REVIEW

File: linebot/models/base.py [hunk 1]
```diff
@@ -78,21 +78,21 @@ def as_json_dict(self):
         :return: dict
         """
         data = {}
-        for key in self.__dict__.keys():
+        for key, sub_obj in self.__dict__.items():
```
Comment (be-hase) [inline linebot/models/base.py]: Thanks ! It became cool.
However, it seems that obj and sub_obj are opposite.

How about changing to `value`, `item`?

```python
        for key, value in self.__dict__.items():
            camel_key = utils.to_camel_case(key)
            if isinstance(value, (list, tuple, set)):
                data[camel_key] = list()
                for item in value:
                    if hasattr(item, 'as_json_dict'):
                        data[camel_key].append(item.as_json_dict())
                    else:
                        data[camel_key].append(item)

            elif hasattr(value, 'as_json_dict'):
                data[camel_key] = value.as_json_dict()

            else:
                data[camel_key] = value

```

Comment (HwangWonYo) [reply linebot/models/base.py]: That's better !

**Comment by HwangWonYo** [comment]
Yes, I submitted the ICLA.

**Comment by be-hase** [comment]
Thanks !

**Pull request merged**